search/search_test.go

```diff
@@ -398,6 +400,34 @@ func TestTarjanSCC(t *testing.T) {
 	}
 }
 
+// batageljZaversnikGraph is the example graph from
```

Comment:
Is it worth having an 'example graphs' package in internal so that way we only need to implement this once?